### PR TITLE
html validation fix

### PR DESCRIPTION
**Pull request by Malposti**
i tag to em change

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -456,7 +456,7 @@ <h1 class="hero__heading">Hi, I'm Michal</h1>
       <section id="skills" class="skills-section">
         <div data-technology-box  class="technology">
           <ul class="technology__list">
-            <li><i class="devicon-html5-plain colored"></i></li>
```
Comment (Malposti) [inline src/index.html:459]: @Malposti looks good

File: src/index.html [hunk 2]
```diff
@@ -456,7 +456,7 @@ <h1 class="hero__heading">Hi, I'm Michal</h1>
       <section id="skills" class="skills-section">
```
Comment (Malposti) [inline src/index.html:456]: @Malposti  can You apply it for all <i> occurances

File: src/index.html [hunk 3]
```diff
@@ -456,7 +456,7 @@ <h1 class="hero__heading">Hi, I'm Michal</h1>
...
         <div data-technology-box  class="technology">
           <ul class="technology__list">
-            <li><i class="devicon-html5-plain colored"></i></li>
+            <li><em class="devicon-html5-plain colored"></em></li>
```
Comment (Postimal) [inline src/index.html:459]: @Malposti looks good, can You apply for all <i> occuracnes

**Review by Postimal — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 3 above)
Comment (Malposti) [reply src/index.html:459]: @Postimal  Done

Comment (Postimal) [reply src/index.html:459]: @Malposti I can notice mistake in list tags, please check it out. Seems that <lem> is not correct

Comment (Malposti) [reply src/index.html:459]: Fixed

**Review by Postimal — APPROVED** [review]

**Pull request merged**